src/nodes/Table/Table.js

```diff
@@ -207,4 +209,25 @@ export default Table.extend({
 		}
 	},
 
+	addProseMirrorPlugins() {
+		return [
+			new Plugin({
+				props: {
+					handlePaste: (view, event, slice) => {
+						if (slice.content.childCount > 1) {
+							const firstNode = slice.content.firstChild
+							let newTextContent = firstNode.textContent
```

Comment:
We'd lose formatting of the node by using its `textContent`, right? Iif formatted text (e.g. bold, italic, ...) gets pasted, that formatting would get lost, no?